docs/manuals/gui/pages/html.md

```diff
@@ -16,12 +16,103 @@ html_page = Html("""
 
 Any <a href="https://en.wikipedia.org/wiki/HTML"><i>HTML</i></a>
 content can be used here.
+
+<taipy:button>Press</taipy:button>
 """)
 ```
 
-The *html_page* variable contains the definition of a page whose content is defined from HTML text.
+The *html_page* variable holds a page whose content is defined from HTML text.
 
 Taipy identifies visual element definitions by finding tags that belong to the `taipy` namespace.
 You can find details on how to create visual elements using HTML in the
 [HTML Syntax](../viselements/index.md#html) section about Visual Elements definition.
 
+# XHTML is required
+
+When Taipy GUI renders the page, it parses the text content to locate the potential visual element
+tags and convert them into new HTML code.<br/>
+This process requires the HTML content to be expressed as [XHTML](https://www.w3.org/TR/xhtml1/).
+XHTML (which stands for "Extensible Hypertext Markup Language") is a markup language that mirrors
+HTML but adheres to the stricter syntax rules of XML (Extensible Markup Language).
+
+Here are the rules that the page definition must follow:
+- The HTML content must be a well-formed XML document (although it does not need a root
+  element).<br/>
+    - Every opening tag must be explicitly closed.<br/>
+      For example, although HTML accepts the tag `<img src="...">`, XHTML does not. You must
+      indicate where the `img` tag ends.<br/>
+      That can be `<img src="..."/>` or `<img src="..."></img>`.
+    - 
+- Strict adherence to syntax rules is required. All elements must be properly nested, closed, and in
+  lowercase.
+    - All the tag names must be lowercase.
+    - All the attribute names must be lowercase, and all values must be quoted.<br/>
+      Note that the XHTML syntax parsed by Taipy GUI in the context of a visual element allows for
+      leaving an attribute valueless. The value for the property is then set to True.
+
+# Event handlers
+
+Taipy GUI drops the event handlers explicitly set using native HTML elements' attributes.
```

Comment:
This sentence doesnt sit right with me. Could not wrap my head around what it is trying to convey.

Reply:
Better in the last commit?